medical_prescription_state/views/medical_prescription_order_line.xml

```diff
@@ -42,6 +42,20 @@
         </field>
     </record>
 
+    <record id="medical_prescription_order_line_form_view_stages" model="ir.ui.view">
```

Comment:
If you're inheriting a view, this id should be the same as that view's id. In this case: `medical_prescription_order_line_view_form`